salt/elife-dashboard/config/srv-app-dashboard-app.cfg

```diff
@@ -0,0 +1,31 @@
+[article_scheduler]
+article_scheduler_url = http://localhost:8001/schedule/v1/article_scheduled_status/
+article_scheduler_publication_url = http://localhost:8001/schedule/v1/schedule_article_publication/
+article_scheduler_range_url = http://localhost:8001/schedule/v1/article_schedule_for_range/from/<from>/to/<to>/
+
+[django]
+debug = true
+allowed_hosts =
+secret_key = (d7lenpn&wwj^t9p!nki60odcw9*8257b1c2+ruj7jrlfehl-o
+
+[database]
+name: dashboard.sqlite3
+engine: django.db.backends.sqlite3
+host =
```

Comment:
This file, being the template for the real `app.cfg`, should use Jinja to print down pillars to build the configuration e.g. `{{ pillar.elife_dashboard_2.db.host }}`

Reply:
Now being populated by pillar values.